### PR TITLE
Worm Component (added hp hooks)

### PR DESCRIPTION
**Pull request by giorgigutsaevi**

### CODE REVIEW

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (kim-morgan) [inline coverage/clover.xml:1]: should add this to gitignore

File: coverage/lcov-report/Worm.js.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
+
```
Comment (kim-morgan) [inline coverage/lcov-report/Worm.js.html:1]: gitignore

File: src/models/Worm.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class Worm {
```
Comment (kim-morgan) [inline src/models/Worm.js:1]: flag for future deletion

File: src/components/Worm.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            top: yBody,
+            width: widthBody,
+            height: heightBody
+        }}/>
+    );
+};
+
+export default (world, color, hp, pos, size) => {
+  const Worm1 = Matter.Bodies.rectangle(
+      pos.x,
```
Comment (AlfonsoGhislieri) [inline src/components/Worm.js:29]: Why is it called Worm1? Would this be a worm instance? Could we have more descriptive name?

**Review by Riky5 — APPROVED** [review]
well done Chris and Gio

**Pull request merged**